src/chrome/background/background.ts

```diff
@@ -36,13 +37,17 @@ const handleOnInstallExtension = async () => {
   }
 }
 
-const handleStorageChange = (changes: {
-  [key: string]: chrome.storage.StorageChange
-}) => {
-  if (changes['connectionPassword'])
-    handleConnectionPasswordChange(changes['connectionPassword']?.newValue)
+const handleStorageChange = (
+  changes: {
+    [key: string]: chrome.storage.StorageChange
+  },
+  area: string,
+) => {
+  if (changes['connections'] && area === 'local') {
+    handleConnectionsChange(changes['connections']?.newValue)
+  }
 
-  if (changes['options']) {
+  if (changes['options'] && area === 'sync') {
```

Comment:
Are we syncing options cross devices?

Reply:
Yes and we always were syncing them. Do you think we shouldn't?

Reply:
I'm not against it but I recall making a decision to keep it local. Reason is that a user can have different wallet setups on different devices. Maybe just relevant for connections.

Reply:
On the other hand you may want to sync your notifications/signaling server settings